Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -14,102 +14,100 @@ See the separate topics for a description and examples of each of these scenario
-    * `onlyApplyOSDPlacement`: Whether the placement specific for OSDs is merged with the `all` placement. If `false`, the OSD placement will be merged with the `all` placement. If true, the `OSD placement will be applied` and the `all` placement will be ignored. The placement for OSDs is computed from several different places depending on the type of OSD:
-        * For non-PVCs: `placement.all` and `placement.osd`
-        * For PVCs: `placement.all` and inside the storageClassDeviceSets from the `placement` or `preparePlacement`
-    * `flappingRestartIntervalHours`: Defines the time for which an OSD pod will sleep before restarting, if it stopped due to flapping. Flapping occurs where OSDs are marked `down` by Ceph more than 5 times in 600 seconds. The OSDs will stay down when flapping since they likely have a bad disk or other issue that needs investigation. If the issue with the OSD is fixed manually, the OSD pod can be manually restarted. The sleep is disabled if this interval is set to 0.
-* `disruptionManagement`: The section for configuring management of daemon disruptions
-    * `managePodBudgets`: if `true`, the operator will create and manage PodDisruptionBudgets for OSD, Mon, RGW, and MDS daemons. OSD PDBs are managed dynamically via the strategy outlined in the [design](https://github.com/rook/rook/blob/master/design/ceph/ceph-managed-disruptionbudgets.md). The operator will block eviction of OSDs by default and unblock them safely when drains are detected.
-    * `osdMaintenanceTimeout`: is a duration in minutes that determines how long an entire failureDomain like `region/zone/host` will be held in `noout` (in addition to the default DOWN/OUT interval) when it is draining. The default value is `30` minutes.
-    * `pgHealthCheckTimeout`: A duration in minutes that the operator will wait for the placement groups to become healthy (see `pgHealthyRegex`) after a drain was completed and OSDs came back up.
-    Operator will continue with the next drain if the timeout exceeds.
-    No values or `0` means that the operator will wait until the placement groups are healthy before unblocking the next drain.
-    * `pgHealthyRegex`: The regular expression that is used to determine which PG states should be considered healthy.
-    The default is `^(active\+clean|active\+clean\+scrubbing|active\+clean\+scrubbing\+deep)$`.
-* `removeOSDsIfOutAndSafeToRemove`: If `true` the operator will remove the OSDs that are down and whose data has been restored to other OSDs. In Ceph terms, the OSDs are `out` and `safe-to-destroy` when they are removed.
-* `cleanupPolicy`: [cleanup policy settings](#cleanup-policy)
-* `security`: [security page for key management configuration](../../Storage-Configuration/Advanced/key-management-system.md)
-* `cephConfig`: [Set Ceph config options using the Ceph Mon config store](#ceph-config)
-* `csi`: [Set CSI Driver options](#csi-driver-options)
+-   `external`:
```

Comment:
In some other docs there was a blank line after the first-level items. Do we not need the blank lines then?

Reply:
My understanding is that blank lines are needed when the type of list changes, like below. But if the type doesn't change, I don't think the spacing is required.

```md
1. one
2. two

    - sub 2.1
    - sub 2.2
    
3. three
```

```md
- first bullet
    
    1. has 1
    2. has 2
    
- next bullet
```